nexus_allowlist/cli.py

```diff
@@ -177,24 +176,17 @@ def initial_configuration(args: argparse.Namespace) -> None:
     # Ensure only desired repositories exist
     actions.recreate_repositories(nexus_api)
 
-    pypi_allowlist, cran_allowlist = actions.get_allowlists(
-        args.pypi_package_file, args.cran_package_file
-    )
-    privileges = actions.recreate_privileges(
-        args.packages, nexus_api, pypi_allowlist, cran_allowlist
-    )
-
```

Comment:
Is this just about simplifying when/where allowlists/privileges are set?

Reply:
Yes, another functions does this already.

It was a bit of a legacy thing where the idea was one command does 'everything' and another just updates the lists.
Now the structure is more like, 'run initial configuration once', 'run allowlist enforcement once, and again whenever you need'.